user/serializers.py

```diff
@@ -2,0 +1,22 @@
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+from user.models import User
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = get_user_model()
+        fields = ("id", "username", "email", "password", "is_staff")
+        extra_kwargs = {"password": {"write_only": True, "min_length": 5}}
+        read_only_fields = ("id", "is_staff")
+
+    def create(self, validated_data):
+        return get_user_model().objects.create_user(**validated_data)
```

Comment:
Ensure that the `create_user` method hashes the password correctly. This method usually handles password hashing, but it's important to verify that the password is not stored in plain text.